test/auto_parallel/pir/test_to_static_pir_program.py

```diff
@@ -66,6 +66,7 @@ def __init__(self, mesh):
         )
 
     def forward(self, x):
+        x.stop_gradient = False
```

Comment:
not need to make x require for gradient, the relu_grad in backward will trigger the  partial-->replicated allreduce